components/mpas-albany-landice/cime_config/buildnml

```diff
@@ -285,6 +285,18 @@ def buildnml(case, caseroot, compname):
 
         lines.append('</stream>')
         lines.append('')
+        if not mali_dynamic:
+            # in MALI data mode, read an input stream with ice thickness monthly
+            lines.append('    <stream name="data-mode-input"')
+            lines.append('            type="input"')
+            lines.append(f'            filename_template="{din_loc_root}/glc/mpasli/{glc_grid}/{grid_prefix}_datamode.nc"')
```

Comment:
I think it would be a good idea to give this file a datestamp.  That way, it could at least be updated and it would be clear which version was expected.